src/binary_helpers.c

```diff
@@ -62,3 +62,54 @@ SR_PRIV int bv_get_value_with_length_check(float *out, const struct binary_value
 		*out = value;
 	return SR_OK;
 }
+
+SR_PRIV int bv_get_value(float *out, const struct binary_value_spec *spec,
+	const void *data)
```

Comment:
Should the void* here also be changed to a uint8_t* as it was for the _with_length_check version?
Currently generates a compiler warning on dereferencing the void pointer..